src/modules/list-item.js

```diff
@@ -0,0 +1,87 @@
+export default class Item {
+  constructor(description, completed = false, index = new Date().getTime()) {
```

Comment:
- [ ] Kindly make sure that the property `index` is set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5).

Reply:
- [ ] As the previous reviewer mentioned please make your new task index equal to the length of the array (i.e. if a 5th task is added to the list, the index of that task should equal to 5).

![image](https://user-images.githubusercontent.com/68796232/166905309-26e5d4be-0e50-49fb-889c-0f0656445f3a.png)